packages/react/src/components/BarChartCard/barChartUtils.js

```diff
@@ -220,7 +220,7 @@ export const formatChartData = (
     series = series.map((item) => omit(item, 'label'));
   }
 
-  let data = values;
+  let data = !isNil(values) ? values : [];
```

Comment:
Can we use the native js `??` here?
```suggestion
  let data =  values ?? [];
```